docker/vernemq/bin/vernemq.sh

```
@@ -0,0 +1,352 @@
+        NODENAME=$(hostname -i)
+        sed -i.bak -r "s/VerneMQ@.+/VerneMQ@${NODENAME}/" ${VERNEMQ_VM_ARGS_FILE}
+    else
+        sed -i.bak -r "s/-name VerneMQ@.+/-name VerneMQ@${IP_ADDRESS}/" ${VERNEMQ_VM_ARGS_FILE}
+    fi
+fi
+
+if env | grep "DOCKER_VERNEMQ_DISCOVERY_NODE" -q; then
+    discovery_node=$DOCKER_VERNEMQ_DISCOVERY_NODE
+    if [ -n "$DOCKER_VERNEMQ_SWARM" ]; then
+        tmp=''
+        while [[ -z "$tmp" ]]; do
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)